src/app/pages/home-page/components/technologies-section/technologies-section.component.ts

```diff
@@ -0,0 +1,41 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {Technology} from '../../../../models/technology';
+import {MatGridListModule} from '@angular/material/grid-list';
+
+@Component({
+  selector: 'app-technologies-section',
+  templateUrl: './technologies-section.component.html',
+  styleUrls: ['./technologies-section.component.css']
+})
+export class TechnologiesSectionComponent implements OnInit {
+  public offSet = 500;
+  technologies: Technology[] = [{
+    id: 0,
+    image: 'assets/angular.jpg',
+    name: 'Angular'
+  }, {
+    id: 1,
+    image: 'assets/javascript.svg',
+    name: 'JavaScript'
+  }, {
+    id: 2,
+    image: 'assets/python.jpg',
+    name: 'Python'
+  }, {
+    id: 3,
+    image: 'assets/android.svg',
+    name: 'Android Development'
+  }];
```

Comment:
open the curlys for the new objects on the new line

{
},
{
},
and add a comma after the last object